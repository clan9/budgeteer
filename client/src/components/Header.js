import React from 'react';
import { Link } from 'react-router-dom';

const Header = () => {
  return (
    <header className="header">
      <div className="content-container">
        <div className="header__content">
          <Link className="header__title" to="/dash">
            <h1>Budgeteer</h1>
          </Link>
          <a href="/auth/logout" className="button button--link">
            Logout
          </a>
        </div>
      </div>
    </header>
  );
};

export default Header;
