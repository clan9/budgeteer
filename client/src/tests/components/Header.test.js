import React from 'react';
import { shallow } from 'enzyme';
import Header from '../../components/Header';

describe('Test Header component', () => {
  it('should render Header component correctly', () => {
    const wrapper = shallow(<Header />);
    expect(wrapper).toMatchSnapshot();
  });
});
