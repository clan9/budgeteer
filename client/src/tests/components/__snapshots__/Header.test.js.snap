// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Header component should render Header component correctly 1`] = `
<header
  className="header"
>
  <div
    className="content-container"
  >
    <div
      className="header__content"
    >
      <Link
        className="header__title"
        to="/dash"
      >
        <h1>
          Budgeteer
        </h1>
      </Link>
      <a
        className="button button--link"
        href="/auth/logout"
      >
        Logout
      </a>
    </div>
  </div>
</header>
`;
